src/templates/plugin.jsx

```diff
@@ -213,6 +221,26 @@ export const pageQuery = graphql`
     jenkinsPlugin(name: {eq: $name}) {
       ...JenkinsPluginFragment
     }
+
+    reverseDependencies: allJenkinsPlugin(
```

Comment:
niiice, no new code elsewhere, just looking at the data we already have, thats awesome.